site/docs/configuration.md

```diff
@@ -22,7 +22,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.dict-size-bytes      | 2097152 (2 MB)     | Parquet dictionary page size                       |
 | write.parquet.compression-codec    | gzip               | Parquet compression codec                          |
 | write.avro.compression-codec       | gzip               | Avro compression codec                             |
-
+| write.metadata.metrics.default     | truncate(16)       | Default metrics mode for all columns in the table; none, counts, truncate(lengthInBytes), or full |
```

Comment:
Truncate length isn't in bytes. It is bytes for binary and characters (unicode codepoints) for UTF-8.

Reply:
I guess I misinterpreted the property name `WRITE_METADATA_TRUNCATE_BYTES`.

Reply:
We probably should have changed that name, good catch.